protoc-gen-grpc-gateway/gengateway/template_test.go

```diff
@@ -383,7 +383,7 @@ func TestApplyTemplateRequestWithClientStreaming(t *testing.T) {
 				},
 			},
 		}
-		got, err := applyTemplate(param{File: crossLinkFixture(&file), RegisterFuncSuffix: "Handler"}, descriptor.NewRegistry())
+		got, err := applyTemplate(param{File: crossLinkFixture(&file), RegisterFuncSuffix: "Handler", AllowPatchFeature: true}, descriptor.NewRegistry())
```

Comment:
Can we add a test case for the new case as well?

Reply:
Sure.

Reply:
Where do you prefer me to add the test? The test infra seems a little bit weak so does it make if i double-check this inside `TestApplyTemplateRequestWithoutClientStreaming`?

Reply:
Could we add a new test case which singles out this case? I'm on mobile ATM so don't have s great view of the files.

Reply:
Done.